src/locale/locales/de/messages.po

```diff
@@ -270,7 +270,7 @@ msgstr "Eine E-Mail wurde an deine vorherige Adresse {0} gesendet. Sie enthält
 
 #: src/lib/moderation/useReportOptions.ts:26
 msgid "An issue not included in these options"
-msgstr ""
+msgstr "Ein Problem, das hier nicht aufgelistet ist."
```

Comment:
```suggestion
msgstr "Ein Problem, das hier nicht aufgelistet ist"
```

I'm not sure what the context of this translation is, but the English translation has no `.` at the end of the sentence, so maybe the actual sentence continues with something else?